resources/prosody-plugins/token/util.lib.lua

```diff
@@ -304,6 +315,10 @@ function Util:process_and_verify_token(session, acceptedIssuers)
             -- Binds any features details to the session
             session.jitsi_meet_context_features = claims["context"]["features"];
           end
+
+          if claims["context"]["tenant"] ~= nil then
```

Comment:
The tenant is already in the 'sub' field, so no need to have a new claim for that part.